### PR TITLE
Fix primary button disabled color and icon button 

### PR DESCRIPTION
**Pull request by mudrila**
## Overview 
Adjust disabled button color for primary button, add styles for icon button, fix disabled color for input

## Changes
- Change `disabled` state color for `primary` `Button`
- Define styles for `IconButton`
- Change `disabled` state color for `Input` to `neutral-6` so that it's brighter than placeholder but still not too bright to visualize different state

## Screenshots
If applicable, provide any necessary screenshots with updates to the UI

## Notes
Design: https://www.figma.com/file/ur5hElxplkmlY9gnGKlwrm/Components-%26-Styles---Decent-Design-System-1.0.0?node-id=288-15125&mode=dev

Related to https://github.com/decentdao/fractal-interface/issues/1693 and https://github.com/decentdao/fractal-interface/issues/1682

## Checklist
- [ ]  Ran all tests and manually tested for bugs
- [ ]  Has tested that app successfully builds/starts locally
- [ ]  Has tested that PR deployment (on develop) has been successful
- [ ]  Call out (review comment) and explain new code changes that may not be self-explainatory (when in doubt, leave a comment)
- [ ]  Call out (review comment) any incomplete code, or code that was added as a shim to complete the PR.
- [ ]  Call out (review comment) code that is hacky or will need to be refactored later.

### CODE REVIEW

**Comment by adamgall** [comment]
I don't really know how to review this code! Would love if @Da-Colon could do a live review with me.

File: design/tokens/components/iconButton/theme/iconButton.sizes.ts [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    height: '1.5rem',
+    width: '1.5rem',
+    padding: '0.25rem',
+  },
+  sm: {
+    apply: 'utility-styles-icon-button-small',
+    height: '1.25rem',
+    width: '1.25rem',
+    padding: '0.25rem'
+  }
```
Comment (dacebt) [inline design/tokens/components/iconButton/theme/iconButton.sizes.ts]: image

The sm icon seems like the padding is off. This was tricky with the IconButton as the width and height properties are for the `Button` with the SVG just being relative to that.

**Review by dacebt — CHANGES_REQUESTED** [review]
The comment applies to all three sizes. @adamgall and I were actually messing with this ealier this evening and found it was much easier to just use the `Icon` as a child to the `Button` than trying to style the `IconButton` component.

We spoke about creating a `icon` size in the `buttons.sizes.ts` to allow for adjustments for the `Button` component with a `Icon` child.

that being said. if you can figure it out how to tweak it to get the padding and size of `IconButton` to match the figma, go for it. ha

**Comment by mudrila** [comment]
@Da-Colon @adamgall 
Here you go 
Знімок екрана 2024-05-11 о 20 38 10

**Comment by mudrila** [comment]
@adamgall If you wanna test these changes - follow these steps

1. Run `nvm use`
2. Run `npm install`
3. Run `npm run storybook`

This will run Storybook on `localhost:6006`, on the left side there will be components list and you can check `IconButton` component.

@Da-Colon @adamgall There's a little hack - I'm not using `padding` in sizes, essentially it's not applied and always overwritten with `padding: 0` for whoknows which reason. So I'm just playing with icon size properly

**Comment by mudrila** [comment]
@adamgall @Da-Colon @DarksightKellar I also pushed a fix for https://github.com/decentdao/fractal-interface/issues/1749 into this branch

**Review by dacebt — APPROVED** [review]
Good deal, LGTM, ship it

**Comment by mudrila** [comment]
Gonna merge now to unblock fixes that relying on these changes

**Pull request merged**